src/components/MAPI/clusters/ClusterDetail/ClusterDetailActions.tsx

```diff
@@ -137,7 +138,27 @@ const ClusterDetailActions: React.FC<
 
   const { canList: canListApps } = usePermissionsForApps(provider, clusterId);
 
-  const appListGetOptions = { namespace: clusterId };
+  const appsNamespace =
+    typeof isClusterApp === 'undefined'
+      ? undefined
+      : isClusterApp
+      ? org?.status?.namespace
```

Comment:
```suggestion
      ? namespace
```
We already have the value of `org?.status?.namespace` assigned to the variable `namespace` on line 65 - do we want to reuse it?

Reply:
good catch